content/en/real_user_monitoring/browser/setup.md

```diff
@@ -17,8 +17,8 @@ The responsibility of keeping user data secure is shared between Datadog and dev
 
 ## Setup
 
-{{< callout url="https://www.datadoghq.com/private-beta/rum-sdk-auto-injection/" btn_hidden="false" header="Try our new Browser SDK injection!">}}
-You can now auto-inject the Browser SDK in your application instead of installing it manually. Complete the form to request access to the private beta.
+{{< callout url="https://www.datadoghq.com/private-beta/rum-sdk-auto-injection/" btn_hidden="true" header="false">}}
+You can now auto-inject the Browser SDK in your application instead of installing it manually. If you're interested in this feature, complete the form to request access.
```

Comment:
Because the button is now hidden, there is no link to the form mentioned in the callout. Suggestion: keep the button visible?

Reply:
Good catch - fixed. Thanks!